### PR TITLE
Sprint 48.5

### PR DESCRIPTION
**Pull request by ericsorenson**
# Sprint 48.5

- Tech Lead: Health and Configuration Endpoints #6281
- Docket Clerks: Sealed Addresses (design + data migration) #6070
- Prod: Configure and Test Email Domain for electronic service #4834

### CODE REVIEW

File: web-client/src/presenter/actions/setMessageAsReadAction.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            docketNumber: '123-45',
+            messageId: '123',
+          },
+        ],
+      },
+    });
+
+    expect(
+      applicationContext.getUseCases().setMessageAsReadInteractor,
+    ).toHaveBeenCalled();
```
Comment (julialeague) [inline web-client/src/presenter/actions/setMessageAsReadAction.test.js]: Should this test that it's been called with the right args?

**Comment by julialeague** [comment]
- [x] If desired, run `docker system prune` to remove any unused docker images from previous code review.
- [x] Fetch the pull request for the sprint (e.g., `git fetch origin pull/{PR #}/head:sprint-{Sprint #}`), and then switch to that branch (e.g. `git checkout sprint-{Sprint #}`)
- [x] review the pull request itself, to get oriented
	- [x] read the description of the pull request, which should summarize the changes made
	- [x] read through every task on the Scrum board that's encompassed by this pull request
	- [x] read the description of the commits that comprise the pull request
- [x] stand up the site locally, with `./docker-run.sh`
	- [x] test all functionality in all major browsers, emphasizing the functionality that this pull request addresses
		- [x] for internal Court functionality, perform the most thorough testing in Chrome, though also test in Edge and Firefox
		- [x] for public-facing functionality, test in browsers consistent with [public browser use data](https://analytics.usa.gov/)
		- [x] test in Mobile Safari and Mobile Chrome (or an emulator like Chrome DevTools), with the caveat that not all internal Court functionality will be necessary on these platforms
	- [x] use an automated audit tool for code quality and practices (recommended: [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/), aka [Lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk))
		- [x] look at efficiency of page loads, asset sizes, HTTP connection management, etc.
	- [x] review for accessibility
		- [x] use an automated audit tool, such as Chrome Audit or aXe
		- [x] navigate site only with the keyboard
		- [x] use VoiceOver or Narrator to navigate the site with audio only, with the display turned off
		- [x] manually test anything that pa11y cannot test automatically (e.g., contrast of text over images)
- [x] [review static code analysis results](https://sonarcloud.io/organizations/flexion-github/projects) in the vendor’s SonarCloud environment
- [x] examine OWASP ZAP output in `docs/`, to ensure that any errors are known to be false positives or have been previously declared to be acceptable
- [x] skim all new code, in the context of existing code, [looking for problems](#what-we-look-for) (knowing that the vast majority of new code will be covered by tests)
- [x] review all tests
	- [x] look at code coverage of tests in [the vendor's SonarCloud environment](https://sonarcloud.io/organizations/flexion-github/projects)
	- [x] methodically review all new tests for correctness, quality of naming
- [x] determine what code isn’t tested, review that rigorously
- [x] review documentation to ensure that it matches changes
- [x] provide comments on the pull request on GitHub, as necessary
	- [x] for comments that are specific to a particular line of code, comment on those specific lines
	- [x] for comments that are more general, attach the comment to a random line in `README.md` (as opposed to commenting on the pull request itself), to be able to use GitHub's ability to thread discussions on those comments
- [x] for each feature-level bug (i.e., it’s working as designed, but designed wrong), open a new issue and put it in the backlog

File: web-client/src/presenter/actions/setMessageAsReadAction.test.js (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/presenter/actions/setMessageAsReadAction.test.js]: Updated!

Comment (julialeague) [reply web-client/src/presenter/actions/setMessageAsReadAction.test.js]: I see this was addressed in https://github.com/ustaxcourt/ef-cms/pull/411/commits/b4abe9f807876323c9ee41e05b324617c0796406, looks good!

**Review by julialeague — APPROVED** [review]

**Comment by julialeague** [comment]
@mmarcotte, I recommend you accept this PR into the court's environments. Thanks!

**Pull request merged**